sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 12.7.0-beta.1 (Unreleased)
+- Added a maxConcurrency option on ParallelTransferOptions that allows the customer to limit how many concurrent network requests will be outstanding at once. 
```

Comment:
One thing that isn't clear is what's the scope of this limit. I believe it's per upload request by looking at implementation. However, one might conclude it's global limit. We should make this clear.

Reply:
Good call. I'll add that to the javadocs, too